assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -363,14 +363,25 @@ export default class LiveEditor {
         settings.editor_mode === "emacs" ? [emacs()] : [],
         language ? language.support : [],
         EditorView.domEventHandlers({
+          click: this.handleEditorClick.bind(this),
           keydown: this.handleEditorKeydown.bind(this),
           blur: this.handleEditorBlur.bind(this),
           focus: this.handleEditorFocus.bind(this),
         }),
+        EditorView.clickAddsSelectionRange.of((event) => event.altKey),
       ],
     });
   }
 
+  /** @private */
+  handleEditorClick(event) {
+    if (event.ctrlKey && event.type === "click") {
```

Comment:
This does not work on Mac, apparently. According to ChatGPT, this is what needs to be written:

```
document.addEventListener('click', function(event) {
    if ((event.metaKey && navigator.platform.toUpperCase().indexOf('MAC') >= 0) || 
        (event.ctrlKey && navigator.platform.toUpperCase().indexOf('MAC') === -1)) {
        console.log('Cmd+Click on Mac or Ctrl+Click on Windows detected');
        // Your logic here
    } else {
        console.log('Regular click detected');
        // Other logic here
    }
});
```

But perhaps we already have something to deal with this (or codemirror).

Reply:
This one:

https://github.com/livebook-dev/livebook/blob/ae5e546bb29b7bf585f0919f27987ad3250c378d/assets/js/hooks/session.js#L299

@aleDsz do we need to check `event.type === "click"`? We already expect it to be a click.

Reply:
CodeMirror have the **[`browser.ts`](https://github.com/codemirror/view/blob/main/src/browser.ts)** file, but they don't export it. Perhaps import the same file to Livebook?

Reply:
I guess we only need this one